DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -102,26 +129,37 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
     private class func statusImageID(connected isConnected: Bool) -> String {
         isConnected ? "VPN" : "VPNDisabled"
     }
-}
 
-private extension ConnectionStatus {
-    var message: String {
-        switch self {
-        case .notConfigured:
-            return "Not Configured"
-        case .disconnected:
-            return "Disconnected"
+    private static func timedConnectedStatusMessagePublisher(forConnectedDate connectedDate: Date) -> AnyPublisher<String, Never> {
+        Timer.publish(every: 1, on: .main, in: .default)
+            .autoconnect()
+            .map {
+                Self.connectedMessage(for: connectedDate, currentDate: $0)
+            }
+            .eraseToAnyPublisher()
+    }
+
+    private static func message(for status: ConnectionStatus) -> String {
+        switch status {
+        case .disconnected, .notConfigured:
+            return UserText.netPStatusDisconnected
```

Comment:
Ohhhhh I commented this on my last review... it's good enough that it's here, no worries about changing this in your other PR.